tests/components/component.jsx

```diff
@@ -1,13 +1,13 @@
-import React from 'react'
+import React, {Component} from 'react'
 import TinyComponent from './tiny-component';
 
-export default class TestComponent extends React.Component {
-  constructor(){
-    super()
+export default class TestComponent extends Component {
+  constructor(...args) {
+    super(...args)
     this.state = {}
   }
 
-  render(){
+  render() {
     return <body>
```

Comment:
If changed to a div, this will break tests in find.jsx. Since this change would be somewhat unrelated to this pull request, maybe @zackify should take a look a this :smile: